bundles/org.openhab.binding.flicbutton/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="flicbutton"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="flicd-bridge">
+		<label>FlicButton Bridge</label>
+		<description>This bridge represents a running instance of the fliclib-linux-hci server (flicd).</description>
+
+		<config-description>
+			<parameter name="hostname" type="text" required="true">
+				<label>Flic daemon (flicd) hostname</label>
```

Comment:
You could specify `<context>network-address</context>` to get a free validation.